### DISCUSSION TITLE
Limit setzen per MQTT/URL/JSON

### DISCUSSION THREAD

**Discussion by RichieXX** [tbnobody/OpenDTU#326] [General]
Ich finde Klasse, dass man mit openDTU auch das Limit des Inverters setzten kann. Geht das auch per MQTT/JSON/URL?

**Comment by tbnobody** [comment]
Sowas steht u.A. in der README:
https://github.com/tbnobody/OpenDTU/blob/8b158a029e94cdf7b44b1404e60b6c860f7f3dd3/README.md?plain=1#L188

**Comment by RichieXX** [comment]
Vielen Dank. Habs hinbekommen. War aber etwas knifflig, das in den iobroker zu bekommen. Hab dann den Datenpunkt mit MQTT Explorer published, dann hat er auch Werte gesendet. Händisches Anlegen im iobroker führt nicht zum Ergebnis.
